Makefile

```diff
@@ -12,7 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+docker/generated.mk: docker/generate_makefile.py $(wildcard fuzzers/*/variants.yaml) install-dependencies
```

Comment:
to fix the infinite loop issue, replace `install-depenedencies` with `${VENV_ACTIVATE}`

Reply:
Thanks! This works.